### PR TITLE
Update mfa-connect-to-exchange-online-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
added the steps to connect to exchange online Public preview which was announced in ignite

### CODE REVIEW

**Comment by yogkumgit** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by webash** [comment]
I've added a slight tweak to the above PR; the instructions don't include a prompt to tell you to do it from an elevated PowerShell window

File: exchange/docs-conceptual/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell.md [hunk 1]
```diff
@@ -133,6 +133,66 @@ If you want to use multi-factor authentication (MFA) to connect to Exchange Onli
 Get-PSSession | Remove-PSSession
 ```
 
+## Connect to Exchange Online PowerShell V2 Module(Public Preview)
+
+1. On your local computer, open Windows PowerShell and run the following command.
```
Comment (JohanFreelancer9) [inline exchange/docs-conceptual/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell.md]:
```suggestion
1. On your local computer, open Windows PowerShell and run the following command:
```

File: exchange/docs-conceptual/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell.md [hunk 2]
```diff
@@ -133,6 +133,66 @@ If you want to use multi-factor authentication (MFA) to connect to Exchange Onli
...
+   Connect-ExchangeOnline -UserPrincipalName chris@contoso.com
+   ```
+
+   This example connects to Exchange Online in Office 365 Germany using the account lukas@fabrikam.com.
+
+   ```PowerShell
+   Connect-ExchangeOnline -UserPrincipalName lukas@fabrikam.com -ConnectionUri https://outlook.office.de/PowerShell-LiveID -AzureADAuthorizationEndPointUri https://login.microsoftonline.de/common
+   ```
+
+   This example connects to Exchange Online to manage another tenant
```
Comment (JohanFreelancer9) [inline exchange/docs-conceptual/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell.md]:
```suggestion
   This example connects to Exchange Online to manage another tenant.
```

**Comment by webash** [comment]
Apparently it also isn't possible to use this module with PowerShell 6 yet, so this should be specified somewhere in the page.

**Comment by webash** [comment]
Here is that PR: https://github.com/msbemba/office-docs-powershell/pull/1

**Comment by msbemba** [comment]
> Here is that PR: [msbemba#1](https://github.com/msbemba/office-docs-powershell/pull/1)

Thank you @webash I'm going to delete the PR you created and add them as commits to my PR. Thanks  again for making our job easy!

**Comment by chrisda** [comment]
@msbemba, we need to add information about the new V2 module in a different way. By essentially repeating all of the download/connection instructions in this topic, it creates problems when the 'real' V2 module topic is updated. Therefore, I'm closing this PR, but I'll incorporate information and a link about the new V2 module near the top of the topic.

**Pull request closed without merging**